src/intervals/queue.jl

```diff
@@ -0,0 +1,72 @@
+# Queue
+# =====
+
+type Queue{T}
```

Comment:
What is the purpose of Queue? Is this Bio specific? If so is there a better name than Queue, which makes one think of DataStructures.jl